src/app/components/typography/TypographyTable.tsx

```diff
@@ -109,14 +109,25 @@ export const TypographyTable = (): JSX.Element => {
         [theme]
     );
 
+    const formatName = useCallback((variant: TypographyVariant): string => {
+        if (variant.startsWith('h')) {
+            const num = variant.charAt(1);
+            if (num === '1') {
+                return `H1`;
+            }
+            return `Headline${num}`;
+        }
+        return variant.charAt(0).toUpperCase() + variant.slice(1);
+    }, []);
+
```

Comment:
Why is H1 not Headline 1 like in Material?

Can we put a space between "Headline" and the number?

<img width="370" alt="image" src="https://user-images.githubusercontent.com/29152776/154972735-89cda054-5a14-4dd0-87a4-2466ac98eb96.png">

Reply:
H1 is not "Headline 1" because the font-size is so big (and name so long) that it causes an overflow in the table which doesn't look so great.  I'll make a small PR to add the space.

Reply:
I abused my admin privs and pushed directly to dev.

Reply:
An overflowing table hides a column which should not be hidden…

And the phrase “headline” also, theoretically speaking, have some alternative meanings in Angular Material.

But, if you insist changing everything to “Headline #”, I suggest taking out the column saying “open sans” as it does not really contribute much information (although I still love seeing all information together)

Reply:
It's not really a wrap, it's an overflow like @huayunh mentioned; maybe we can talk about this in office hours and figure out a good solution.  I'll wait til we have consensus before making any code changes.

Reply:
If I change H1 to Headline 1, i see a wrap, not an overflow (no hidden columns):
<img width="1144" alt="image" src="https://user-images.githubusercontent.com/29152776/155160408-ae86c5ec-1678-4d21-a07b-f405736737d0.png">

And you'll run into overflow on smaller screens even if you call it H1...that's unavoidable with tables.

Headline 1 is (~90%) less confusing than H1/H2/H3/H4 because there is no `<headline>` element that you can mistakenly use. Thinking that using an `<h1>` will get you a BLUI Headline 1 style in Angular Material is what sparked this issue to begin with.

What about something like:
<img width="804" alt="image" src="https://user-images.githubusercontent.com/29152776/155161432-5c2ac975-b7bd-404e-ae18-5a80a76cc454.png">


I'm also okay with dropping the typeface column as a quick workaround for now.

Reply:
^ Can do this for a smaller view port